petl/test/io/test_remotes.py

```diff
@@ -3,11 +3,16 @@
 
 import sys
 import os
+import time
```

Comment:
`time` is not used, is it?

Reply:
No.
Also flake8 was complaining about it!